lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/grok/Grok.java

```diff
@@ -119,14 +129,23 @@ public Grok()
         originalGrokPattern = "";
         namedRegex = "";
         compiledNamedRegex = null;
-        grokPatternDefinition = DEFAULT_PATTERNS;
-        grokPatternDefaultDatatype = DEFAULT_DATA_TYPES;
+        grokPatternDefinition = new HashMap<>(DEFAULT_PATTERNS);
```

Comment:
Was this supposed to be a `TreeMap`? If so, then `DEFAULT_PATTERNS` will also need to use `ImmutableSortedMap`